Sources/Main.cpp

```diff
@@ -177,6 +178,100 @@ void TX584Form::LoadPRJ(TFileStream *Stream)
         CodeListView->Items->Item[i]->SubItems->Strings[3] = L"";
     }
 }
+
+void TX584Form::LoadX584(TFileStream *Stream)
```

Comment:
This branch also contains erroneous copies of `LoadX584` and `LoadPRJ`. Please get rid of them.